src/main/java/com/iota/iri/service/dto/IXIResponse.java

```diff
@@ -15,4 +17,24 @@ public static IXIResponse create(Object myixi) {
     public Object getResponse() {
         return ixi;
     }
+
+    private String getdefaultContentType() {
+        return "application/json";
+    }
+
+    public String getResponseContentType() {
+        Object fieldObj = getResponseMapper().get("contentType");
+        String fieldValue = fieldObj == null || "".equals(fieldObj) ? getdefaultContentType() : fieldObj.toString();
+        return fieldValue;
+    }
+
+    private Map<String, Object> getResponseMapper(){
+        return (Map<String, Object>)ixi;
+    }
+
+    public String getContent() {
+        Object fieldObj = getResponseMapper().get("content");
+        String fieldValue = fieldObj == null || "".equals(fieldObj) ? null : fieldObj.toString();
```

Comment:
Should `"   "` pass?

Use `StringUtils.isNotEmpty()` or `StringUtils.isNotBlank()` according to your answer